src/core/models.py

```diff
@@ -1420,9 +1424,17 @@ class Meta:
 class EditorialGroup(models.Model):
     name = models.CharField(max_length=500)
     description = models.TextField(blank=True, null=True)
+    press = models.ForeignKey(
+        'press.Press',
+        on_delete=models.CASCADE,
+        blank=True,
+        null=True,
+    )
```

Comment:
Shouldn't this be required too? We can provide it a default to ensure it is set correctly:
```
def default_press():
    return Press.objects.first()
[...]
press = models.ForeignKey(
        'press.Press',
        on_delete=models.CASCADE,
        blank=True,
        null=True,
       default=default_press,    
```